scss/menu.scss

```
@@ -0,0 +1,126 @@
+.menu__nav {
+  position: fixed;
+  left: 5%;
+  top: var(--nav-height);
+  width: 90%;
+  box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
+  border-radius: 10px;
+  overflow: hidden;
+  background-color: var(--container-color);
+  transform: translateY(-365px);
+  transition: 0.3s;
+}
+
+.menu__nav.active {
+  align-items: center;
+  transform: translateY(10px);
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  align-items: center;
+  overflow: hidden;
+  background-color: var(--container-color);
+}
+
+.navigation__li {
+  margin: 15px 0;
+  transition: 0.2s;
+}
+
+.navigation__li:hover {
+  transform: scale(1.1, 1.1);
+}
+
+.navigation__a {
+  text-decoration: none;
+  color: var(--title-color);
+}
+
+.navigation__a:hover {
+  color: var(--first-color);
+}
+
+.navigation__li:nth-of-type(1) {
+  margin-top: 30px;
+}
+
+.navigation__li:nth-child(4) {
+  margin-bottom: 30px;
+}
+
+.navigation {
+  position: fixed;
```

```suggestion
  position: sticky;
```

is a better solution

---

Alright